src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -44,6 +45,7 @@
     @Value("${storage.blob_lease_timeout}")
     private Integer blobLeaseTimeout;
 
+    @Autowired
```

Comment:
Is this required? Spring `@Component`s should be automatically autowired

Reply:
Not required, but I find it easier to read when it's explicit. I can follow your convention, that's not a problem.

Reply:
If it makes faster to read code and moreover - application to boot up - I'm more than happy to go back and start using this extra annotation